src/main/kotlin/dev/stalla/util/LocaleExtensions.kt

```diff
@@ -0,0 +1,21 @@
+package dev.stalla.util
+
+import java.util.Locale
+
+/**
+ * Maps all available [Locale] language tags to their [Locale] instance.
+ *
+ * @see Locale.getAvailableLocales
+ * @see Locale.toLanguageTag
+ */
+@InternalApi
+internal val localeMap: Map<String, Locale> = Locale.getAvailableLocales().associateBy({ it.toLanguageTag() }, { it })
+
+/**
+ * Returns `true` when this [Locale]'s language tag is defined in the available locales.
+ *
+ * @see Locale.getAvailableLocales
+ * @see Locale.toLanguageTag
+ */
+@InternalApi
+internal fun Locale.isValidLocale(): Boolean = localeMap[this.toLanguageTag()] != null
```

Comment:
Test cases showed that `java.util.Locale` seems to allow parsing from nonsense values like `xyz`. The class is weird for [historic reasons](https://softwareengineering.stackexchange.com/questions/325458/parse-both-en-us-and-en-us-as-locale-in-java/367706#367706). Fortunately, the workaround to only accept defined values is pretty simple.

Reply:
I think inline classes are beta atm. I'll give them a try.

Reply:
So inline classes were not able to have an `init` block for [Java interop reason](https://github.com/Kotlin/KEEP/blob/master/proposals/inline-classes.md#overloads-private-constructors-and-initialization-blocks), but since Kotlin 1.4.30 this is now possible. 

However, the only real thing we can do here is to `require(locale.isValidLocale())` which raises an exception then of course. Technically this could be ok for the RSS language, because it must be valid. The Feedpress property on the other hand is irrelevant of course and just requires us to catch the exception in the first place. But I don't see how we can `null` the argument with this approach if it's invalid. Or am I missing something obvious here? (It's been a long day, so I wouldn't be surprised 😅).  

I don't like the idea of refusing to parse a feed due to an invalid language. Clients like podcatchers directory/indizes will still want to parse the feed, even if the language is null. This is something more suited for our [ModelValidator](https://github.com/mpgirro/stalla/issues/46) feature here.

Reply:
I think you're right. I have had some very busy days and I hadn't really considered the context, here, either 😅 So, what we have is a-ok, unless we figure out some better way to do it. Once we get validators in place, that'll be the best way to achieve it.